Automattic/wp-calypso
# Gutenboarding: move design picker state in onboarding store

#### Changes proposed in this Pull Request

* add selected page layouts to onboarding store
* <s>[WIP] add selected design to store</s> Already [implemented]( https://github.com/Automattic/wp-calypso/pull/38705/files#diff-b43d337b3bf37258ebc7f894a3e9b6a5L48-R48) in https://github.com/Automattic/wp-calypso/pull/38705.

#### Testing instructions
* Selected page layouts should be persisted during one visit (including when selecting a different design or navigating to previous step and back)

Fixes one issue mentioned in https://github.com/Automattic/wp-calypso/pull/38583#pullrequestreview-337251522
> Esc cancels all page selections and returns to design picker, just like clicking outside page selector does as well. I expected my page selections to remain tho; they got wiped out.
